### PR TITLE
Refactor dashboard components structure

### PR DESCRIPTION
**Pull request by rawagner**
- Move all dashboards under components/dashboard
 - Move generic dashboard components to @console-shared

FYI I moved `OWNERS` file from `public/components/dashboard` to `@console-shared/src/components/dashboard` too

### CODE REVIEW

**Comment by christianvogt** [comment]
Or `@console/dashboard`

**Comment by rawagner** [comment]
> How about moving the generic dashboard components into `@console/shared/src/components/dashboard`?

moved to `console-shared`. @christianvogt Im not sure what you mean by `@console/dashboard`.

**Comment by jtomasek** [comment]
> > How about moving the generic dashboard components into `@console/shared/src/components/dashboard`?
> 
> moved to `console-shared`. @christianvogt Im not sure what you mean by `@console/dashboard`.

I think he simply means moving it into a separate plugin structurally similar to @console/shared.

**Comment by wking** [comment]
/hold

Merge conflicts.  Wave off the retest bot until this gets rebased.

**Comment by vojtechszocs** [comment]
> Merge conflicts. Wave off the retest bot until this gets rebased.

Will rebase, along with follow-up #2895.

**Comment by vojtechszocs** [comment]
> FYI I moved `OWNERS` file from `public/components/dashboard` to `@console-shared/src/components/dashboard` too

@spadgett FYI

File: frontend/packages/console-shared/src/styles/skeleton-screen.scss [hunk 1]
```diff
@@ -0,0 +1,2 @@
+$skeleton-animation: loading-skeleton 1s ease-out .15s infinite alternate;
+$skeleton-color: #eaedf1;
```
Comment (vojtechszocs) [inline frontend/packages/console-shared/src/styles/skeleton-screen.scss]: Nit: add trailing newline.

**Comment by vojtechszocs** [comment]
Rebased & addressed the nit comment, linter & tests should pass.

**Comment by vojtechszocs** [comment]
@wking OK to un-hold?

**Comment by vojtechszocs** [comment]
The only effective change (beyond the rebase conflict resolution) is newline addition in `skeleton-screen.scss`, so I can re-add the LGTM label once we un-hold the PR.

**Comment by jelkosz** [comment]
PR has been un-hold. Based on Vojtech's comment this only waits for the formal lgtm label. Giving it:

/lgtm

**Pull request merged**